pytorch_lightning/trainer/training_loop.py

```diff
@@ -203,7 +197,7 @@ def on_train_epoch_start(self, epoch):
         model = self.trainer.get_model()
 
         # reset train dataloader
-        if self.trainer.reload_dataloaders_every_epoch:
+        if epoch != 0 and self.trainer.reload_dataloaders_every_epoch:
```

Comment:
```suggestion
        if epoch > 0 and self.trainer.reload_dataloaders_every_epoch:
```
it is there a case with negative value?

Reply:
I don't think there would an instance of negative values.